### PR TITLE
Extended Taxiviewer to mostly eliminate hand-editing of JSON files.

### PR DESCRIPTION
**Pull request by EyeMWing**

### CODE REVIEW

File: TaxiViewer/MainWindow.xaml.cs [hunk 1]
```diff
@@ -306,26 +335,47 @@ private void MouseUpHandler(object s, MsaglMouseEventArgs ev)
                     .Intersect(TargetNode.Node.Id.Replace('-', ' ').Split())
                     .FirstOrDefault();
 
+                if(taxiName == null || Int32.TryParse(taxiName, out _))
+                {
+                    taxiName = SourceNode.Node.Id.Replace('-', ' ')
+                    .Split()
+                    .Intersect("Alpha Bravo Charlie Delta Golf Echo Foxtrot Hotel India Juliett Juliet Kilo Lima Mike November Oscar Papa Quebec Romeo Sierra Tango Uniform Victor Whiskey Yankee Zulu".Split())
```
Comment (rurounijones) [inline TaxiViewer/MainWindow.xaml.cs]: You are going to want `Alfa` in here as well. Some people use that. (and it is apparently the proper spelling of the phonetic A, I was suprised too. the https://en.wikipedia.org/wiki/NATO_phonetic_alphabet#/media/File:FAA_Phonetic_and_Morse_Chart2.svg )

File: TaxiViewer/MainWindow.xaml.cs [hunk 2]
```diff
@@ -306,26 +335,47 @@ private void MouseUpHandler(object s, MsaglMouseEventArgs ev)
...
+                    .Split()
+                    .Intersect("Alpha Bravo Charlie Delta Golf Echo Foxtrot Hotel India Juliett Juliet Kilo Lima Mike November Oscar Papa Quebec Romeo Sierra Tango Uniform Victor Whiskey Yankee Zulu".Split())
+                    .FirstOrDefault();
+                }
+
+                if (taxiName == null || Int32.TryParse(taxiName, out _))
+                {
+                    taxiName = TargetNode.Node.Id.Replace('-', ' ')
+                    .Split()
+                    .Intersect("Alpha Bravo Charlie Delta Golf Echo Foxtrot Hotel India Juliett Juliet Kilo Lima Mike November Oscar Papa Quebec Romeo Sierra Tango Uniform Victor Whiskey Yankee Zulu".Split())
```
Comment (rurounijones) [inline TaxiViewer/MainWindow.xaml.cs]: Ditto above. Maybe make this a constant

**Review by rurounijones — CHANGES_REQUESTED** [review]
Niiiiiiiiiiiiiiiiiiice, one minor comment.

File: TaxiViewer/MainWindow.xaml.cs (hunk 1 above)
Comment (EyeMWing) [reply TaxiViewer/MainWindow.xaml.cs]: Best part: I was literally reading off that chart to begin with. Done, done and also added support for double-letter taxiways

File: TaxiViewer/MainWindow.xaml.cs (hunk 2 above)
Comment (EyeMWing) [reply TaxiViewer/MainWindow.xaml.cs]: See above

**Pull request merged**